README.md

```diff
@@ -37,3 +37,25 @@ your prompt should look similar to this this:
 Run the tests on your machine:
 
     pytest -vv digicampipe
+
+## Pointing using lid CCD images
+
+To be able to determine the pointing using stars reflections on the lid, you need to install astrometry.net:
+
+    sudo apt-get install astrometry.net
+    
+You will need to download catalogs too: 
+
+    sudo mkdir ~/astrometry-data
```

Comment:
I think you do not need to use `sudo` to make directory in your home, unless it is important that this directory belongs to the `root`. Is that important?